tools/gdal/gdal_translate.xml

```diff
@@ -18,7 +55,73 @@
             <when value="simple">
             </when>
             <when value="advanced">
-				<param name="b" type="text" label="-b band" help="Select an input band band for output. Bands are numbered from 1. Multiple -b switches may be used to select a set of input bands to write to the output file, or to reorder bands." value="-b 1 -b 2 -b 3"/>
+            	<param name="strict" type="boolean" truevalue="-strict" falsevalue="" checked="false" label="-strict" help="Don't be forgiving of mismatches and lost data when translating to the output format."/>
```

Comment:
The indentation should be 4 spaces instead of a tab.

Reply:
Thks for the corrections !
Why should we prefer 4 spaces to a tab ?

Reply:
It is just a best practice recommendation. No strict need, but we try all to stict to some standards to have a unified look and feel.

Also many more recommendations here: http://galaxy-iuc-standards.readthedocs.io/en/latest/index.html

Reply:
Ok i'll try to force myself to use that standard, and i'll take a look at the iuc recommendations